Pareto/NavigationUI/matRad_plotPS.m

```diff
@@ -0,0 +1,25 @@
+function matRad_plotPS(helperObj)
```

Comment:
This function was initially for being able to show 3D surfaces within the GUI navigator and where the current plan is located there. Rewriting this so it can be called from outside to visualize 2D/3D tradeoffs. 
Maybe will later add an option for using it from the GUI (with some checkboxes to select which objective to use)